drag_and_drop_v2/translations/eo/LC_MESSAGES/text.po

```diff
@@ -582,8 +582,12 @@ msgstr ""
 "séléçt ä dïfférént ïtém). Ⱡ'σяєм ιρѕυм ∂σłσя ѕιт αмєт, ¢#"
 
 #: public/js/drag_and_drop.js
-msgid "OK"
-msgstr "ÖK Ⱡ'σя#"
+msgid "Close"
+msgstr "Çlösé Ⱡ'σя#"
```

Comment:
@cahrens I'm not sure what you mean? It is not empty in this file. It is empty in the english text.po file, but all msgstrs are empty in the english text.po file.